site/en/guide/dtensor_overview.ipynb

```diff
@@ -98,7 +100,7 @@
       },
       "outputs": [],
       "source": [
-        "!pip install --quiet --upgrade --pre tensorflow"
+        "!pip install --quiet --upgrade tensorflow"
```

Comment:
We usually don't bother with installing tensorflow. It was only here because the pre-installed version was insufficient.